plasmapy/simulation/particle_integrators.py

```diff
@@ -0,0 +1,76 @@
+import math
+import numpy as np
+
+from astropy import constants
+
+
+def boris_push(x, v, b, e, q, m, dt):
+    r"""
+    Implement the explicit Boris pusher for moving and accelerating particles.
+
+    Parameters
+    ----------
+    x : np.ndarray
+        particle position at full timestep
+    v : np.ndarray
+        particle velocity at half timestep
+    b : np.ndarray
+        magnetic field at full timestep
+    e : float
+        electric field at full timestep
```

Comment:
I'm wondering about adding some more detail to the arguments.  Reading this, I could end up being confused about the `ndarray` objects...like if they are three-vectors corresponding to x, y, z.  I could also get confused about what units are assumed.